tests/test_qb_timeline.py

```diff
@@ -190,6 +193,131 @@ def test_completed_input(self):
         assert QBT.query.filter(QBT.status == OverallStatus.in_progress).one()
         assert QBT.query.filter(QBT.status == OverallStatus.completed).one()
 
+    def test_consent_change(self):
+        # Basic w/ one complete QB followed by consent change
+        # should clear QNR->QB relationships
+        crv = self.setup_org_qbs()
+        org_id = crv.id
+        back7, nowish = associative_backdate(
+            now=datetime.utcnow(), backdate=relativedelta(days=7))
```

Comment:
[non-determinism in tests is non-good](https://martinfowler.com/articles/nonDeterminism.html)